src/pages/Admin/Venue/VenueAdminPage.scss

```diff
@@ -1,4 +1,17 @@
-.admin-page-title {
-  text-align: center;
-  padding-top: 25px;
+@import "scss/constants";
+
+.AdminPage {
+  position: relative;
+  max-width: 638px;
+  margin: 0 auto;
+
+  &__title {
+    text-align: center;
+    margin-top: 41px;
+    padding: 5px;
+    font-size: $font-size--sm;
+    background: #1a1d24;
```

Comment:
From consts. If there is no such const, it should be created

Reply:
https://github.com/sparkletown/sparkle/pull/1440/commits/395b101c6f7eab7bd705293241f259ed46d806ff